python/tests/test_chatcompletion_mock_wrapper.py

```diff
@@ -0,0 +1,25 @@
+import lastmile_utils.lib.core.api as cu
+
+
+class MockRealCompletions:
+    create = "real"
+
+
+class MockRealChat:
+    completions = MockRealCompletions()
+
+
+class MockRealOpenAI:
+    """
+    We have to mock the real openAI to test the actual mock because the real openai needs an API key.
+    """
+
+    some_real_field: str = "the_real_value"
+    chat = MockRealChat()
```

Comment:
What's the use for this? I'm assuming tests be added ontop

Reply:
I had to mock the real object so I could test the real mock. lol

Otherwise the test can't run because it doesnt have the API key.